src/wallet/wallet.cpp

```diff
@@ -3517,43 +3497,48 @@ bool CWallet::HasCollateralInputs(bool fOnlyConfirmed) const
     return !vCoins.empty();
 }
 
-bool CWallet::CreateCollateralTransaction(CMutableTransaction& txCollateral, std::string& strReason)
+bool CWallet::CreatePrivateSendCollateralTransaction(CMutableTransaction& txCollateral, std::string& strReason)
```

Comment:
Why such as long name? to me CreateCollateralTransaction is good

Reply:
There is also budget/governance collateral though. Maybe `CreatePrivateSendCollateralTX` or `CreatePSCollateralTransaction` then?

Reply:
Thoughts about 82614d633bb7422b83c8050be761ee344e01b077?